src/pages/AdminDashboard/data.js

```diff
@@ -0,0 +1,129 @@
+// import { v4 as uuidv4 } from "uuid";
```

Comment:
I'll go over on Monday how we can convert this file to use in our request mocking.